dallinger/config.py

```diff
@@ -32,6 +32,7 @@
     ('aws_region', unicode, []),
     ('aws_secret_access_key', unicode, [], True),
     ('base_payment', float, []),
+    ('blacklist', unicode, []),
```

Comment:
Maybe we should call it `qualification_blacklist` in case some other kind of blacklist needs to be configured in the future?